sdk-python/omnilogger/pricing/pricing_class.py

```diff
@@ -0,0 +1,110 @@
+import logging
+import os
+
+from omnilogger.pricing.storage_manager import (
+    copy_json,
+    pricing_config_path,
+    pricing_reference_path,
+    read_json_at,
+)
+from omnilogger.pricing.typedDicts import ModelData
+
+
+class Pricing:
+    data: dict[str, dict[str, ModelData]]
+    help_edit_msg: str
+
+    def __init__(self, is_test: bool = False):
```

Comment:
- If is_test is used to point the pricing.json path to a different location for testing, I would avoid including it in the API 
- Instead, I would mock the path